themes/bootstrap3/js/common.js

```diff
@@ -665,6 +709,9 @@ $(function commonDocReady() {
   // handle QR code links
   setupQRCodeLinks();
 
+  // Setup facet list autocomplete
+  setupFacetList();
```

Comment:
An easy way is to inline the script in the template loaded in lightbox. This also ensures that the script runs regardless of whether the page is displayed in lightbox or standalone. There are several places where this is used, just search the code for `lightbox compatibility`.

Reply:
It worked, thanks!

Reply:
However, it still seems like the `more` button doesnt work properly. I guess due to the changed URL lookups it doesn't work anymore. I will investigate further when time permits; however i would be very thankful for more hints related to this issue.

Reply:
I think i finally found a way how to fix the `more` button! As suspected it was necessary to combine the existing lightbox facet functionality with the new ajax mechanics. It works now, but still needs some optimization (maybe it would be better to move the entire code in facet_list.js to facets.js anyway).

Reply:
Glad to hear you're making progress, @mtrojan-ub! I'll try to take a closer look at this next week when I'm back from vacation if nobody else gets there first. :-)

Reply:
That would be very helpful!

In the meantime i added some more bugfixes / usability improvements and also merged things back together (e.g. facet_list.js got merged into the existing facets.js and the AJAX handler got merged into the existing AbstractList action). I still wonder whether we could improve the code even more, e.g. by encapsulating the FacetList-related JS functions into a separate namespace similar to sideFacets, or by reducing the amount of passed parameters for the AJAX handling).